src/doing/utils.py

```diff
@@ -286,16 +286,16 @@ def guess_shell_encoding() -> str:
         return sys.stdout.encoding
 
 
-def verbose_shell():
+def verbose_shell() -> bool:
     """
     If shell commands should be printed.
 
     Users can define 'verbose_shell: True' in the .doing-cli-config.yml file.
     """
-    return get_config("verbose_shell", fallback=False)
+    return get_config("verbose_shell", fallback=False)  # type: ignore[no-any-return]
```

Comment:
Similar issue here. We could add an assertion that `get_config` returns a `bool`, but I opted to add typing with minimal runtime overhead by adding an ignore.